scripts/analyze_failed_jobs/action_plugins/parsestdouts.py

```diff
@@ -0,0 +1,50 @@
+import re, glob, bz2
+from ansible.utils import template
+from ansible.runner.return_data import ReturnData
+
+IGNORED_BUILDERS = [
+    'flocker',
+    'flocker-twisted-trunk', 
+    'flocker-acceptance-vagrant-centos-7-zfs',
+    'flocker-installed-package-vagrant-centos-7'
+]
+
+class ActionModule:
+    def __init__(self, runner):
+        self.runner = runner
+        
+        
+    def run(self, conn, tmp, module_name, module_args, inject, complex_args=None, **kwargs):
```

Comment:
I could document though the Ignored Builders constant... although its pretty self-explanatory.
To make it "proper" I should have used an Ansible parameter, but you requested a quick and dirty hack